src/js/components/stock-movement-wizard/request/CreateStockMovement.jsx

```diff
@@ -198,6 +231,14 @@ class CreateStockMovement extends Component {
     if (this.state.values.origin && this.state.values.destination) {
       this.fetchStockLists(this.state.values.origin, this.state.values.destination);
     }
+    if (this.state.values.origin) {
+      if (this.state.values.origin?.supportedActivities?.includes(ActivityCode.APPROVE_REQUEST)) {
```

Comment:
what about destination's supportedActivities? Has it been clarified what we should look at exactly?